test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -1855,8 +1855,6 @@ public async Task RestoreCommand_LockedLockFileWithOutOfDateProjectAsync()
             }
         }
 
-#if IS_DESKTOP
-        // TODO: To work on coreclr we need to address https://github.com/NuGet/Home/issues/7588
```

Comment:
I think .NET Core solved this `problem`. 
If the OS supports it ofc. 

The max path length is now shortmax but there might be restrictions about certain segments. 

tldr; 

I don't think you can get it to throw on .NET Core if the machine has long path support enabled (automatic on latest windows versions)

Reply:
I've pushed a new change that skips testing this if long paths are enabled on the current machine. Does that seem like the right way to go about this?

Reply:
Yeah, I think that's ok. 

Should this change be targeting dev instead of the feature branch?